source/default_mode/Symbiont.h

```diff
@@ -388,26 +388,17 @@ class Symbiont: public Organism {
     }
   }
 
-
-  /**
-   * Input: The double representing the resources to be distributed to the symbionts
-   *
-   * Output: The double representing the host's resources
-   *
-   * Purpose: To process and distribute resources.
-  */
-  double ProcessResources(double hostDonation){
-    return ProcessResources(hostDonation, my_host);
-  }
-
   /**
-   * Input: The double representing the resources to be distributed to the symbionts and the host from whom it comes
+   * Input: The double representing the resources to be distributed to the symbionts and the host from whom it comes; if no host is provided it will automatically use my_host
```

Comment:
Change to:

Input: The double representing the resources to be distributed to the symbionts and (optionally) the host from whom it comes; if no host is provided, the symbiont's host variable is used.